internal/tofu/transform_provider.go

```diff
@@ -613,7 +771,7 @@ type ProviderConfigTransformer struct {
 	// each provider node is stored here so that the proxy nodes can look up
 	// their targets by name.
 	providers map[string]GraphNodeProvider
-	// record providers that can be overridden with a proxy
+	// record providers that can be overriden with a proxy
```

Comment:
I don't think it's a typo
```suggestion
	// record providers that can be overridden with a proxy
```

Reply:
https://github.com/opentofu/opentofu/pull/1963/commits/93aaf45228ccf0e9341cf03727b505249eeb2f1c